src/test/java/api/requests/scenarios/MoveRequestTests.java

```diff
@@ -932,8 +945,8 @@ public void canUpdateSourceAndDestinationLoanDueDateOnMoveRecallRequest() throws
         .equals(interestingTimesItem.getId().toString()))
       .findFirst().orElse(new JsonObject());
 
-    assertThat(smallAngryPlanetLoan.getInstant("dueDate"), is(expectedLoanDueDate));
-    assertThat(interestingTimesLoan.getInstant("dueDate"), is(expectedLoanDueDate));
+    assertThat(smallAngryPlanetLoan.getInstant("dueDate"), is(expectedJamesLoanDueDate));
```

Comment:
Why are the due dates set to the system date and time? Is that because the policy being used has a zero recall return interval?

Reply:
This was an existing test. I assume the default loan policy being applied to the tests doesn't have a minimum guaranteed due date duration, so we apply the recall with the system date here, which is the default behavior. In the recall tests I wrote I supplied a loan policy with explicit intervals.

Reply:
I understand it is an existing test and does not fall fully within the scope of this work. I only noticed it due to the changes. 

Maybe it is out of scope of this change to improve this test?

Reply:
Do you think we need explicit policies set here? I didn't think there was anything wrong with the test, I just needed to make it work since the expectations had changed.

Reply:
I found it hard to follow why the due date was meant to be the system time. (And maybe the policy that if no recall return interval is set, it expects immediate return, which seems odd to me in general).

I mostly just didn’t understand, let’s defer this to another time